deploy/compose/hoprd.cfg.yaml

```diff
@@ -2,48 +2,22 @@
 hopr:
   host:
     address: !IPv4 127.0.0.1 # NOTE: adapt this to reflect your public IP
-    port: 9091 # NOTE: adapt this to reflect your public port
+    port: 9091 # NOTE: adapt this to reflect your public p2p port
   db:
     data: /app/hoprd/
-    initialize: true
-    force_initialize: false
   strategy:
     on_fail_continue: true
     allow_recursive: true
     strategies:
       - !Aggregating
-        aggregation_threshold: 100
+        aggregation_threshold: 330
         unrealized_balance_ratio: 0.95
         aggregate_on_channel_close: true
       - !AutoRedeeming
         redeem_only_aggregated: true
-        on_close_redeem_single_tickets_value_min: "90000000000000000 HOPR"
+        on_close_redeem_single_tickets_value_min: "2000000000000000000 HOPR"
       - !ClosureFinalizer
         max_closure_overdue: 60
-  heartbeat:
```

Comment:
By removing these people will have to consult what the config options are with the docs.

Reply:
There was an error in the previous config file content, and I made it work by setting only the required attributes.
I prefer to set less information duplicated, because later we will need to take care of updating them as well.